app/controllers/UploadController.scala

```diff
@@ -36,7 +36,7 @@ class UploadController @Inject()(val controllerComponents: SecurityComponents,
         case Some("Completed") =>
           uploadStatus match {
             case Some("InProgress") =>
-              Ok(views.html.uploadInProgress(consignmentId, pageHeading, request.token.name)).uncache()
+              Ok(views.html.uploadInProgress(consignmentId, pageHeading, request.token.name, request.token.isJudgmentUser)).uncache()
```

Comment:
This is wrapped in the `standardTypeAction` so this will always be false. The line below hard codes false in for isJudgmentUser so it might be worth doing that here.

Reply:
good point